### PR TITLE
First draft May newsletter

### PR DESCRIPTION
**Pull request by bethaitman**
Ready for review!

### CODE REVIEW

File: docs/blog/newsletter-may-2019.rst
Comment (hillaryfraley) [inline docs/blog/newsletter-may-2019.rst]: We're also delighted... (add "also" maybe?)

File: docs/blog/newsletter-may-2019.rst
Comment (hillaryfraley) [inline docs/blog/newsletter-may-2019.rst]: ...save it for posterity! (add "it")

File: docs/blog/newsletter-may-2019.rst
Comment (hillaryfraley) [inline docs/blog/newsletter-may-2019.rst]: "community" is repeated at the end of this intro sentence

File: docs/blog/newsletter-may-2019.rst
Comment (hillaryfraley) [inline docs/blog/newsletter-may-2019.rst]: hyphenate "time-consuming" maybe?

File: docs/blog/newsletter-may-2019.rst
Comment (hillaryfraley) [inline docs/blog/newsletter-may-2019.rst]: again hyphenate "higher-quality" as a modifier?

File: docs/blog/newsletter-may-2019.rst
Comment (hillaryfraley) [inline docs/blog/newsletter-may-2019.rst:58]: love this heading!

File: docs/blog/newsletter-may-2019.rst
Comment (hillaryfraley) [inline docs/blog/newsletter-may-2019.rst]: I think "in" is redundant with "substitute" -- "...substitute their own data."

File: docs/blog/newsletter-may-2019.rst
Comment (hillaryfraley) [inline docs/blog/newsletter-may-2019.rst]: bit odd to say the report found -- maybe "One survey of articles and forum posts found that..." instead?

File: docs/blog/newsletter-may-2019.rst
Comment (hillaryfraley) [inline docs/blog/newsletter-may-2019.rst]: "whereas" instead of "where"?

File: docs/blog/newsletter-may-2019.rst
Comment (hillaryfraley) [inline docs/blog/newsletter-may-2019.rst:92]: :D love it!

File: docs/blog/newsletter-may-2019.rst
Comment (clairealyssa) [inline docs/blog/newsletter-may-2019.rst]: Do we want to somehow say here that it brings it to consideration for the newsletter? Things marked with the emoji may not be 1:1 with newsletter articles.

File: docs/blog/newsletter-may-2019.rst
Comment (clairealyssa) [inline docs/blog/newsletter-may-2019.rst]: Should this be a colon or a comma? Just a small nitpick.

File: docs/blog/newsletter-may-2019.rst
Comment (clairealyssa) [reply docs/blog/newsletter-may-2019.rst]: Even though I originally wrote it without the hyphen, looking at it here, I have turned team pro-hyphen.

**Review by clairealyssa — COMMENTED** [review]
Thank you, Beth!

File: docs/blog/newsletter-may-2019.rst
Comment (bethaitman) [reply docs/blog/newsletter-may-2019.rst]: agreed!

File: docs/blog/newsletter-may-2019.rst
Comment (bethaitman) [reply docs/blog/newsletter-may-2019.rst]: fixed!

File: docs/blog/newsletter-may-2019.rst
Comment (bethaitman) [reply docs/blog/newsletter-may-2019.rst]: +1, changed

File: docs/blog/newsletter-may-2019.rst
Comment (bethaitman) [reply docs/blog/newsletter-may-2019.rst]: agreed, taken it out

File: docs/blog/newsletter-may-2019.rst
Comment (bethaitman) [reply docs/blog/newsletter-may-2019.rst]: :+1:

File: docs/blog/newsletter-may-2019.rst
Comment (bethaitman) [reply docs/blog/newsletter-may-2019.rst]: 👍

File: docs/blog/newsletter-may-2019.rst
Comment (bethaitman) [reply docs/blog/newsletter-may-2019.rst]: Yeah, might be worth making it clear that we don't write up everything. I'll adjust.

File: docs/blog/newsletter-may-2019.rst
Comment (bethaitman) [reply docs/blog/newsletter-may-2019.rst]: changed

File: docs/blog/newsletter-may-2019.rst
Comment (plaindocs) [inline docs/blog/newsletter-may-2019.rst]: > to the nearer future

 this reads weird at first glance.

**Comment by bethaitman** [comment]
Still waiting on an approval, if anyone cares to?

**Review by plaindocs — APPROVED** [review]
Excellent work!

File: docs/blog/newsletter-may-2019.rst
Comment (bethaitman) [reply docs/blog/newsletter-may-2019.rst]: I can rephrase

**Pull request merged**
